File: chapter2.page23.classes/src/main/java/j/Person.java
package j;

/**
 * @author PaulFrmBrn
 */
// value object concept - only data with no code
public class Person {
    private final String name;

    public Person(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
}

File: chapter2.page23.classes/src/main/kotlin/k/Person.kt
package k

/**
 * @author PaulFrmBrn
 */
// value object concept - only data with no code
// default visibility - public
// properties in Kotlin is combination of Java's fields and accessors
class Person(val name: String, // read-only property - field and getter
             var isMarried: Boolean // modifiable property: field + setter + getter
             )
// Custom accessors can be used